lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -4614,11 +4610,7 @@ public SecurityAdvice isAllowed(String userId, String function, String reference
 		
 		// Create or update a task
 		String reference = "/lessonbuilder/page/" + page.getPageId();
-		if (this.createTask) {
```

Comment:
Could we add a comment here?
// Dashboard task widget: When a page item is added create a task item for the site members.